Views/Tag-TermPart.liquid

```diff
@@ -1,13 +1,16 @@
 ﻿{% assign termName = Model.ContentItem | display_text %}
 
+{% assign taggedContentItems = Queries["ContentByTags"] | query: tagId: Model.ContentItem.ContentItemId %}
```

Comment:
Can you see if the following works, would save a unnecessary variable declaration.

```
{% assign taggedContentItemsCount = Queries["ContentByTags"] | query: tagId: Model.ContentItem.ContentItemId | size %}
```